src/python/tests/core/bot/testcase_manager_test.py

```diff
@@ -758,6 +760,7 @@ def setUp(self):
     """Set up."""
     super(UntrustedEngineReproduceTest, self).setUp()
     environment.set_value('JOB_NAME', 'libfuzzer_asan_job')
+    environment.set_value('USE_MINIJAIL', False)
```

Comment:
@oliverchang - fyi, there is some flakiness in CI here. USE_MINIJAIL=True was carried behind from corpus pruning untrusted test. i could not find root cause, i see patch_environ there. locally it does not fail.